arm/xarm.go

```diff
@@ -293,7 +293,7 @@ func NewXArm(ctx context.Context, name resource.Name,
 
 	err = x.start(ctx)
 	if err != nil {
-		return nil, multierr.Combine(errors.Wrap(err, "failed to start xarm"), err, x.Close(ctx))
+		logger.Warnf("the xArm couldn't be started because: %s clear the error status before issuing command to the arm", err)
```

Comment:
this is interesting that we do not return here, but I guess we will fail immediately below when we call the current inputs method.